pkg/plugins/resources/postgres/store.go

```diff
@@ -267,3 +274,94 @@ func (r *resourceMetaObject) GetCreationTime() time.Time {
 func (r *resourceMetaObject) GetModificationTime() time.Time {
 	return r.ModificationTime
 }
+
+func registerAndCollectMetrics(metrics core_metrics.Metrics, dbStats sql.DBStats) error {
```

Comment:
Again - you are only collecting the initial metrics about the DB. You need to do `db.Stats()` inside the closures.

Also change the name for `registerMetrics`